kedro/extras/datasets/pickle/pickle_dataset.py

```diff
@@ -51,6 +51,11 @@
 except ImportError:  # pragma: no cover
     joblib = None
 
+try:
+    import compress_pickle
```

Comment:
Can you also update the docstrings on line 31 please? :)

Reply:
Done! Thank you.